pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -88,31 +86,19 @@ Use '--force' to skip the prompt confirming if you want to uninstall Velero.
 }
 
 // Run removes all components that were deployed using the Velero install command
-func Run(ctx context.Context, kbClient kbclient.Client, namespace string, waitToTerminate bool) error {
-	var errs []error
-
-	// namespace
-	ns := &corev1.Namespace{}
-	key := kbclient.ObjectKey{Name: namespace}
-	if err := kbClient.Get(ctx, key, ns); err != nil {
-		if apierrors.IsNotFound(err) {
-			fmt.Printf("Velero namespace %q does not exist, skipping.\n", namespace)
-		} else {
-			errs = append(errs, errors.WithStack(err))
-		}
-	} else {
-		if ns.Status.Phase == corev1.NamespaceTerminating {
-			fmt.Printf("Velero namespace %q is terminating.\n", namespace)
-		} else {
-			if err := kbClient.Delete(ctx, ns); err != nil {
-				errs = append(errs, errors.WithStack(err))
-			}
-		}
+func Run(ctx context.Context, kbClient kbclient.Client, namespace string) error {
+	// The CRDs cannot be cleaned up before the namespace being deleted to avoid the issue https://github.com/vmware-tanzu/velero/issues/3974
```

Comment:
Grammar rewrite:
The CRDs cannot be removed until the namespace is deleted to avoid the problem in issue https://github.com/vmware-tanzu/velero/issues/3974 so if the namespace deletion fails we error out here